# Service to load the light positions
string lps
---
bool successfully_loaded